### DISCUSSION TITLE
Not working after generating APK.

### DISCUSSION THREAD

**Discussion by AndresdoSantos** [nativewind/nativewind#491] [Q&A]
I have an app running correctly on the emulator, but when I generate an APK the app breaks all styling.

I'm using expo-router, I have pages with () and with [...].

My `tailwind.config.js` looks like this: 

```
module.exports = {
  content: ['./app/*.tsx', './app/**/*.tsx', './components/*.tsx'],
  theme: {
    extend: {
      fontFamily: {
        'inter-400': ['Inter_400Regular'],
        'inter-500': ['Inter_500Medium'],
      },
      fontSize: {
        '2xs': '9px',
      },
    },
  },
  plugins: [],
}
```

I really don't know what it could be.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
